examples/cardreader_diagnostics.json

```diff
@@ -2,6 +2,7 @@
   "name": "Card Reader Diagnostics",
   "payload": {
     "eventTimestamp": "2022-05-24T07:00:25Z",
+    "traceId": "3841a268-0c03-4588-b476-211be0f26a0d",
```

Comment:
Hvis dette skal endres, må VIX inn i bildet. Det er de som har software for dette. However, det bør kanskje sjekkes om noen virkelig er interessert i disse meldingene. Det var Tommy Degerdalen m.fl. som i sin tid ønsket å få dette innført, og Martin Johnsen kommuniserte med VIX for å få det implementert. Jeg samler opp meldingene og dytter dem på Kafka, men såvidt jeg vet er det ingen som konsumerer dem. De bør enten brukes til noe eller så bør dette fjernes.

Reply:
@mariusw tenker det må være opp til dere om dere ønsker dette eller ikke. Jeg kan godt reversere denne endringen hvis det ikke er ønskelig å involvere tredjeparter i dette arbeidet nå.

Reply:
Rullet tilbake endring